emily/handler/src/api/routes/mod.rs

```diff
@@ -11,13 +11,30 @@ mod chainstate;
 mod deposit;
 /// Health routes.
 mod health;
+/// Testing routes.
+#[cfg(feature = "testing")]
+mod testing;
 /// Withdrawal routes.
 mod withdrawal;
 
 /// This function sets up the Warp filters for handling all requests.
+#[cfg(feature = "testing")]
 pub fn routes(
     context: EmilyContext,
 ) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+    // TODO(273):  Remove the "local" prefix once we figure out why all local
+    // testing calls seem to forcibly start with `local`.
+    warp::path("local").and(
+        health::routes()
+            .or(chainstate::routes(context.clone()))
+            .or(deposit::routes(context.clone()))
+            .or(withdrawal::routes(context.clone()))
+            .or(testing::routes(context)),
+    )
+}
+
+#[cfg(not(feature = "testing"))]
+fn debug_routes() -> Option<impl Filter<Extract = impl warp::Reply, Error = warp::Rejection>> {
```

Comment:
Shouldn't this be `routes` with the other one called `debug_routes`?

Reply:
fixing!

Reply:
This was entirely wrong; both should have been called `routes` and had the same function signature.